### PR TITLE
[Central Beds] Fetch updates directly from Symology API

### PR DESCRIPTION
**Pull request by davea**
The CSVs don't include everything we need to generate updates for FixMyStreet, so this PR calls into the Symology API for each enquiry in the CSV file and builds the Open311 ServiceRequestUpdates from the Symology history events.

### CODE REVIEW

File: perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm [hunk 1]
```diff
@@ -47,7 +47,10 @@ sub _event_status {
 
     my $map = $self->endpoint_config->{event_status_mapping}->{$event->{HistoryType}};
     return unless $map;
-    return $map->{$event->{HistoryEventType}};
+    return $map unless ref $map eq 'HASH';
+    my $field = $map->{field};
+    $map = $map->{values};
+    return $map->{$event->{$field}};
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm:53]: Could the event_status_mapping be given example/documentation in the yml example file.
Also this code doesn't seem to match with the config currently on staging?

File: perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm [hunk 2]
```diff
@@ -126,14 +126,17 @@ sub _update_for_history_event {
     my $status = $self->_event_status($event);
     return unless $status;
     my $description = $self->_event_description($event);
-
     my $update_id = $crno . '_' . $event->{LineNo};
+    my $external_status = $event->{HistoryType};
+    $external_status = "${external_status}_" . $event->{HistoryEventType} if $external_status eq '21';
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm:135]: Could "21" be explained?

**Review by dracos — APPROVED** [review]
Looks good, would be good to have all the configuration parameters documented/exampled.

File: perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm (hunk 1 above)
Comment (davea) [reply perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm]: Oops, yes, staging was out of date. Have fixed that now, and added docs in ce010a06a5614b3be92679733daf7cd6d24e3549.

File: perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm (hunk 2 above)
Comment (davea) [reply perllib/Open311/Endpoint/Integration/UK/CentralBedfordshire.pm]: I'll go one better - it's removed in eb1dca284ea0817186fe7ab588b978f3699f4dfc.

**Pull request merged**